ai-service/tests/conftest.py

```
@@ -0,0 +1,32 @@
+    return client.get_or_create_collection(collection_name)
+
+
+@pytest.fixture(scope="module")
+def db_test_collection(request) -> chromadb.Collection:
+    collection_name = f"{request.module.__name__}"
+    return create_db_test_collection(collection_name)
+
+
+@pytest.fixture(autouse=True)
+def patch_and_clean_db_collection(
+    monkeypatch, db_test_collection
```

Ruff (0.12.2)

Missing type annotation for function argument `monkeypatch`

(ANN001)